### PR TITLE
Lesson 3 - homework - buttons

### PR DESCRIPTION
**Pull request by anadanielkiewicz**
I am kindly asking you to check my work. I will be grateful for the time devoted to my code. I tried to create different buttons and also create those from the lessons. I also focused a lot on css and code form. For example, based on the Good & Bad button, I figured out how to arrange the buttons on one line in the menu. I will still practice them so that I can write code from my head.

### CODE REVIEW

File: homework_lesson_3/style_lesson_3_buttons.css [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    text-align: center;
+    padding: 14px 16px;
+    text-decoration: none;
+    font-size: large;
+    font-weight: bold;
+    float: right;
+    list-style-type: none;
+}
+
+.HowAreYouButton{
```
Comment (aberezhk) [inline homework_lesson_3/style_lesson_3_buttons.css]: camel case will be .howAreYouButton (starts with small letter)

File: homework_lesson_3/index_lesson_3_buttons.html [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+    <nav class="main_navigation">
+        <ul>
+            <li><a href="#Home">Home</a></li>
+            <li><a href="#Aboutme">About me</a></li>
+            <li><a href="#Basepage" class="active">Base page</a></li>
+            
+            <!-- JavaScript Nav Button-->
+            <div class="buttonNav">
+              <button id="NavButtonOne"> Click me</button>
+              <button id="NavButtonTwo"> Feel good button </button>
```
Comment (aberezhk) [inline homework_lesson_3/index_lesson_3_buttons.html]: Feel good button should have been added together with good/bad button. Could you please move it there? Please take a look at the example here: https://test-redi-js1.github.io/redi_js_spring_2020/buttons/. The idea was to have burger button (collapsed menu icon - https://en.wikipedia.org/wiki/Hamburger_button) to the menu. But you can skip this part and keep your burger button as it is now.

File: homework_lesson_3/index_lesson_3_buttons.html [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+
+<!-- JavaScript Good & Bad Button-->
+  <div class="HowAreYouButton">
+
+    <p>How are you?</p>
+    <button id="goodButton">Good 👍</button>
+    <button id="badButton">Bad 👎</button>
+    <p id="answer"></p>
+
+    <script>
```
Comment (aberezhk) [inline homework_lesson_3/index_lesson_3_buttons.html]: There is no need to have multiple script blocks in one html file. You can put all the js code in one script block.

File: homework_lesson_3/index_lesson_3_buttons.html [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+    </script>
+  </div> <br><br>
+
+ <!-- JavaScript Burger Button-->
+<div>
+  <a id="burgerButton" href="#"> </a>
+  
+  <script type="text/javascript">
+  
+  const burgerButtonEl=document.getElementById('burgerButton')
```
Comment (aberezhk) [inline homework_lesson_3/index_lesson_3_buttons.html]: Do not forget about ;

**Review by aberezhk — COMMENTED** [review]
Good work on buttons. Funny solution of burger button =) Could you please move feel awesome button next to good and bad button, make sure that all variable and class names start with small letters and add semicolons? The burger button will be accepted as it is.

File: homework_lesson_3/style_lesson_3_buttons.css (hunk 1 above)
Comment (anadanielkiewicz) [reply homework_lesson_3/style_lesson_3_buttons.css]: Of course It is now clear for me, I will start from small letter. Thank you for pointing out an error to me.

File: homework_lesson_3/index_lesson_3_buttons.html (hunk 2 above)
Comment (anadanielkiewicz) [reply homework_lesson_3/index_lesson_3_buttons.html]: Sorry, I did not know it should looks like that. I changed my code adding awesome buttom  below next to good&bad buttons. Thank you for the explanation.

File: homework_lesson_3/index_lesson_3_buttons.html (hunk 3 above)
Comment (anadanielkiewicz) [reply homework_lesson_3/index_lesson_3_buttons.html]: I removed js code part from nav place ( now it is unnecesary). But before to have those buttons in navigation menu I thought i have to place them just here - am I sure? Please tell me if i think wrong.

File: homework_lesson_3/index_lesson_3_buttons.html (hunk 4 above)
Comment (anadanielkiewicz) [reply homework_lesson_3/index_lesson_3_buttons.html]: I will do my best not to forget about semicolons :) Thank you!

**Review by aberezhk — APPROVED** [review]
Please merge to master

**Pull request merged**